### PR TITLE
Fix Pandas Warning

### PR DESCRIPTION
**Pull request by colin99d**
# Description

Fix the Pandas warning from the issue I posted. Unfortunately, I just had to mute the warning. See this StackOverflow issue: https://stackoverflow.com/questions/20625582/how-to-deal-with-settingwithcopywarning-in-pandas.

# How has this been tested?

Reran the table with no warning showing.

# Checklist:

- [x] Update [our Hugo documentation](https://gamestonkterminal.github.io/GamestonkTerminal/) following [these guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/website).
- [x] Update our tests following [these guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/blob/main/CONTRIBUTING.md).
- [x] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/scripts).

# Others
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My code passes all the checks pylint, flake8, black, ... To speed up development you should run `pre-commit install`.
- [x] New and existing unit tests pass locally with my changes. You can test this locally using `pytest tests/...`.

### CODE REVIEW

**Review by jmaslek — APPROVED** [review]

File: gamestonk_terminal/stocks/discovery/ark_view.py [hunk 1]
```diff
@@ -65,15 +65,15 @@ def ark_orders_view(
         df_orders = df_orders[df_orders.direction == "Buy"]
```
Comment (northern-64bit) [inline gamestonk_terminal/stocks/discovery/ark_view.py:65]: The correct way to solve the warnings is to add .copy() to every slice here:
```suggestion
        df_orders = df_orders[df_orders.direction == "Buy"].copy()
```

File: gamestonk_terminal/stocks/discovery/ark_view.py [hunk 2]
```diff
@@ -65,15 +65,15 @@ def ark_orders_view(
...
     if sells_only:
         df_orders = df_orders[df_orders.direction == "Sell"]
```
Comment (northern-64bit) [inline gamestonk_terminal/stocks/discovery/ark_view.py:67]: Add .copy() here too

**Review by northern-64bit — CHANGES_REQUESTED** [review]
The warning comes from making df_orders a slice of df_orders and then making a slice of this slice. Pandas doesn't like this, so one must add .copy() to the first slices so it is a df thus not making a slice of slice. This has to be added to line 63, 65, 67 and maybe some more that I have missed.

File: gamestonk_terminal/stocks/discovery/ark_view.py (hunk 1 above)
Comment (colin99d) [reply gamestonk_terminal/stocks/discovery/ark_view.py]: Thanks for the input! I was actually able to triangulate the issue to the ark_model. Somewhere in the code surrounded by the pandas warning mute functions. I tried adding this code to those but the warning still appeared. Do you have any further suggestions?

Comment (northern-64bit) [reply gamestonk_terminal/stocks/discovery/ark_view.py]: I don't know the error here, but it's probably an error of the same type or due to using df[value][value2] insted of iloc or loc. The error is probably here I'm however not sure (line 110-113 or maybe in 102-103 due to not using iloc and loc):
bild

**Review by northern-64bit — APPROVED** [review]
I sadly can't find the cause of the warning and this is a good solution.

**Pull request merged**